src/lang/modifyAst/transforms.spec.ts

```diff
@@ -396,7 +314,11 @@ scale(extrude001, factor = 2)`
         instanceInThisFile,
         kclManagerInThisFile
       )
-      const nodeToEdit = createPathToNodeForLastVariable(ast, false)
+      const nodeToEdit: PathToNode = [
+        ['body', ''],
+        [3, 'index'],
+        ['expression', 'ExpressionStatement'],
+      ]
```

Comment:
This was the real fix! Of course we didn't want a path to node to the last variable 🤦 

We only went into recast territory deep in getNodeFromPath cause it was failing